src/routes/cache-hooks/cache-hooks.service.ts

```diff
@@ -130,6 +130,14 @@ export class CacheHooksService {
             chainId: event.chainId,
             safeAddress: event.address,
           }),
+          this.safeRepository.clearMultisigTransactions({
+            chainId: event.chainId,
+            safeAddress: event.address,
+          }),
+          this.safeRepository.clearTransfers({
+            chainId: event.chainId,
+            safeAddress: event.address,
+          }),
```

Comment:
Clearing both `MultisigTransactions` and `Transfers` in several hooks defined in this class, since the PR changes expect to have these endpoints cache clearer when an event happens in order to get the last timestamp.